charms/reactive/bus.py

```diff
@@ -65,18 +75,19 @@ class Handler(object):
     _CONSUMED_FLAGS = set()
```

Comment:
Not sure why you want _CONSUMED_FLAGS here instead of _consumed_flags. All-caps is generally reserved for constants in Python. Similarly, _HANDLERS below.